assignments/arrays.js

```diff
@@ -63,7 +63,12 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+for(let i = 0; i === inventory["id" : "33"; i++){
```

Comment:
In Challenge 1, it wants you to replace the *car year goes here*, *car make goes here*, and *car model goes here*. In layman term, you want to call the variable, and select the object with an id of 33, and output the name it wants in each spots.